android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/components/BottomScreenSection.kt

```diff
@@ -51,7 +51,7 @@ fun BottomScreenSection(
   ) {
     val navBackStackEntry by navController.currentBackStackEntryAsState()
     val currentRoute = navBackStackEntry?.destination?.id
-    mainNavigationScreens.filter { it.showInBottomNav }.forEach { navigationScreen ->
+    mainNavigationScreens.asSequence().filter { it.showInBottomNav }.forEach { navigationScreen ->
```

Comment:
I do not think there will be any performance improvement using a sequence here for just the `filter` and `forEach` operations.